### ISSUE TITLE
Build images with GitHub Actions

### ISSUE THREAD

**Issue by neugartf** [thyrlian/AndroidSDK#54]
We could leverage GitHub Actions to build the images directly here and push them to the registries.

**Comment by thyrlian** [comment]
Agree, I also thought about getting rid of the free Travis CI build plan.

**Comment by thyrlian** [comment]
GitHub Actions [CI workflow](https://github.com/thyrlian/AndroidSDK/actions) has been added.

Just need to save DockerHub credentials to actions secrets, and create one more step for pushing images.

But I'm considering using other tags than `latest` for the GitHub Actions automatically pushed images, because they're not properly verified (to be working).
